app/models/item.rb

```diff
@@ -2,6 +2,7 @@ class Item < ApplicationRecord
   validates :name, presence: true
   validates :status, presence: true
 
-  belongs_to :user
+  belongs_to :user, class_name: 'User'
```

Comment:
Why do we have to define the `class_name` here? Are you planning to add another model association for the `User` model?